api/src/main/java/org/openmrs/module/isanteplusreports/library/queries/ReportQueries.java

```diff
@@ -0,0 +1,33 @@
+package org.openmrs.module.isanteplusreports.library.queries;
```

Comment:
@hmutaquiha please remove this whole class since it's meant to be used as an example